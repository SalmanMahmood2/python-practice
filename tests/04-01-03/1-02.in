remove
350z
print
