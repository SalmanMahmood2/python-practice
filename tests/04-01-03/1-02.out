S14
240z
R33